astropy/coordinates/tests/test_angles.py

```diff
@@ -1009,3 +1009,13 @@ def test_longitude_nan():
 def test_latitude_nan():
     # Check that passing a NaN to Latitude doesn't raise a warning
     Latitude([0, np.nan, 1] * u.deg)
+
+
+def test_angle_multithreading():
+    import  threading
```

Comment:
Could you add a comment with a link back to the issue here? For future selves, it would be particularly useful to know that the loop over multiple threads would fail if the angle parser was not thread-save.

Reply:
Also, just move the `import threading` to the top (and one space, not two)